docs-mslearn/fabric/create-finops-workspace.md

```diff
@@ -0,0 +1,235 @@
+    3. Subscription = *Contoso subscription*
+    4. Storage account = *Contoso-storage-account*
+    5. Container = *msexports*
+    6. Directory = focuscost/&lt;*scope-id*&gt;
+    7. Overwrite data = **Enabled**
+5. On the Review + Create tab:
+    - Select **Create**
+
+Ensure that your export completed and that the file is available before moving to the next step.
+
+>[!IMPORTANT]
+> Cost Management exports utilize managed identity. To create an export, you must be an Owner of the storage account or have access to grant permissions to the storage account. To see all required permissions, see [Prerequisites](/azure/cost-management-billing/costs/tutorial-improved-exports#prerequisites).
+
+If you want to automate export creation, consider using the [New-FinOpsCostExport command](https://microsoft.github.io/finops-toolkit/powershell/cost/New-FinOpsCostExport) in the FinOps toolkit PowerShell module.
+
+If you deployed [FinOps hubs](https://aka.ms/finops/hubs), you can use the exports configured against the hub.
+
+## Create a Fabric workspace
+
+1. Sign in to the Fabric app using your [Fabric free account](https://www.microsoft.com/microsoft-fabric/getting-started) or [Microsoft Power BI](https://app.powerbi.com) account.
+2. You can create a new workspace for your FinOps resources or choose to use an existing workspace. To create a workspace, see [Create a workspace](/fabric/get-started/create-workspaces).
+
+If you already have a [Microsoft Fabric workspace](/fabric/get-started/workspaces), you can skip creating a new workspace, unless you want to keep your FinOps resources separate from others.
+
+If you didn't set up Microsoft Fabric, see [Enable Microsoft Fabric for your organization](/fabric/admin/fabric-switch).
+
+If you’re new to Microsoft Fabric or don’t have Fabric capacity enabled, see [Fabric trial](/fabric/get-started/fabric-trial). If Fabric trials are disabled, you might need to [create a new Fabric capacity](/fabric/admin/capacity-settings?tabs=fabric-capacity#create-a-new-capacity).
+
+Nontrial Fabric capacity incurs charges for your organization. See [Microsoft Fabric pricing](https://azure.microsoft.com/pricing/details/microsoft-fabric/).
+
+Here's an example screenshot showing a new Fabric workspace getting created.
+
+:::image type="content" source="./media/create-fabric-workspace-finops/fabric-create-workspace.png" border="true" alt-text="Screenshot showing a workspace getting created." lightbox="./media/create-fabric-workspace-finops/fabric-create-workspace.png" :::
+
+## Create a lakehouse
+
+You can either create a new lakehouse or use an existing one. Lakehouses are interoperable so we would generally recommend using a single lakehouse for all your FinOps datasets. A single lakehouse keeps related information together to share data models, pipelines, and security measures.
+
+To create a new lakehouse in the Fabric workspace of your choice:
+
+1. Select **New** > **More options**.  
```

Comment:
Do we need to set context on where they go to select this option?

Reply:
Updated